.coveragerc

```diff
@@ -548,7 +548,7 @@ omit =
     homeassistant/components/sochain/sensor.py
     homeassistant/components/socialblade/sensor.py
     homeassistant/components/solaredge/sensor.py
-    homeassistant/components/solax/sensor.py
```

Comment:
why you remove solax?

Reply:
hmm.. Sorry about that.  I'm not used to the rebase workflow and must have messed something up.